src/frontends/pytorch/src/op/avg_poolnd.cpp

```diff
@@ -40,35 +40,16 @@ OutputVector translate_avg_poolnd(const NodeContext& context) {
     }
     ov::op::RoundingType rounding_type = ov::op::RoundingType::FLOOR;
     if (!(context.input_is_none(4))) {
-        rounding_type = context.const_input<bool>(4) ? ov::op::RoundingType::CEIL : ov::op::RoundingType::FLOOR;
+        rounding_type = context.const_input<bool>(4) ? ov::op::RoundingType::CEIL_TORCH : ov::op::RoundingType::FLOOR;
     }
     if (!(context.input_is_none(5))) {
         count_include_pad = context.const_input<bool>(5);
     }
     PYTORCH_OP_CONVERSION_CHECK(context.input_is_none(6),
                                 "Translation for aten::avg_pool2d do not support divisor_override input.");
-    // Although ov::AvgPool provides exclude_pad=false,
-    // The corner case of Average Pooling with ceil_mode on
-    // PyTorch allows sliding window go off bound, which leads to this accommodation.
-    // More detail on https://github.com/pytorch/pytorch/issues/57178
-    if (count_include_pad) {
-        auto zero = context.mark_node(v0::Constant::create(element::f32, Shape{}, {0}));
-        zero = context.mark_node(std::make_shared<v1::ConvertLike>(zero, input));
-        auto zero_i32 = context.mark_node(v0::Constant::create(element::i32, Shape{}, {0}));
-        Output<Node> rank;
-        std::tie(std::ignore, rank) = get_shape_rank(context, input);
-        auto pad_values = context.get_input(3);
-        auto pads_len = context.mark_node(v0::Constant::create(element::i32, Shape{}, {pads.size()}));
-        auto pads_diff = context.mark_node(std::make_shared<v1::Subtract>(rank, pads_len));
-        auto pads_remaining = context.mark_node(std::make_shared<v3::Broadcast>(zero_i32, pads_diff));
-        auto padding = context.mark_node(
-            std::make_shared<v0::Concat>(OutputVector{std::move(pads_remaining), std::move(pad_values)}, 0));
-        input = context.mark_node(std::make_shared<v1::Pad>(input, padding, padding, zero, ov::op::PadMode::CONSTANT));
-        pads = Shape(pads.size(), 0);
-    }
 
     return {context.mark_node(
-        std::make_shared<v1::AvgPool>(input, strides, pads, pads, kernel, !count_include_pad, rounding_type))};
+        std::make_shared<v14::AvgPool>(input, strides, pads, pads, kernel, !count_include_pad, rounding_type))};
```

Comment:
Do we have downgrading transformation for lowering version of `AvgPool` and `MaxPool`? That is to support plugins without v14 imlementation?